applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.cpp

```diff
@@ -173,6 +174,31 @@ Vector SmallStrainUPwDiffOrderElement::GetPressures(const size_t n_nodes) const
     return pressure;
 }
 
+void set_arithmetic_average_pressure(Geometry<Node>&                               rGeometry,
```

Comment:
I had one more suggestion about the naming of this function, not a blocker, but you could do it if you would like to pick up any of the other suggestions

```suggestion
void SetArithmeticAveragePressure(Geometry<Node>&                               rGeometry,
```